webpack.dev.config.js

```diff
@@ -29,7 +22,7 @@ const ethereumUrl =
 module.exports = {
   context: path.join(__dirname, 'src'),
   entry: ['react-hot-loader/patch', 'bootstrap-loader', 'index.js'],
-  devtool: 'eval',
+  devtool: 'source-map',
```

Comment:
eval-source-map may be a better fit. It is faster on rebuild and used for development only.